packages/appstore/src/components/modals/wallet-modal/wallet-modal.tsx

```diff
@@ -1,19 +1,44 @@
 import React from 'react';
 import { Modal } from '@deriv/components';
+import { useCurrencyConfig, useWalletList } from '@deriv/hooks';
+import { localize } from '@deriv/translations';
 import WalletModalHeader from './wallet-modal-header';
 import WalletModalBody from './wallet-modal-body';
 import { observer, useStore } from '@deriv/stores';
```

Comment:
@heorhi-deriv a suggestion to reorder the import 🙏.

Reply:
sorry @aum-deriv maybe i'm not fully understand how to sort the imports, i checked the documentation but maybe i'm not fully realise how to sort them